### PR TITLE
Add profile template inclusion tags

### PR DESCRIPTION
**Pull request by phlax**
to separate out logic/templates in user/profile.html

### CODE REVIEW

File: pootle/templates/user/includes/profile_score.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+<h2 class="user-score-title" style="display: inline">{% trans 'Total score:' %}</h2>
+<h3 class="user-score" style="display: inline">
+  {% if score_tweet_link %}
+  <a class="js-popup-tweet"
+     href="{{ score_tweet_link }}"
+     title="{% blocktrans %}Tweet this!{% endblocktrans %}">{{ profile.scores.public_score }}</a>
```
Comment (unho) [inline pootle/templates/user/includes/profile_score.html]: Nit: Use simpler `{% trans 'Tweet this!' %}` instead

File: pootle/templates/user/includes/profile_teams.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% if is_superuser %}
+<h3>Site administrator</h3>
```
Comment (unho) [inline pootle/templates/user/includes/profile_teams.html]: Not translatable?

File: pootle/templates/user/includes/profile_teams.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+{% endif %}
+{% if teams %}
+<h3 style="margin-top: 1em">Language teams</h3>
```
Comment (unho) [inline pootle/templates/user/includes/profile_teams.html]: Make this translatable.

File: pootle/templates/user/includes/profile_score.html (hunk 1 above)
Comment (phlax) [inline pootle/templates/user/includes/profile_score.html]: ill move it to inclusion tag

**Comment by unho** [comment]
Nits aside, lgtm.

File: pootle/templates/user/includes/profile_teams.html (hunk 2 above)
Comment (phlax) [inline pootle/templates/user/includes/profile_teams.html]: hmm - im not sure whether we want to keep it at all - but ill move it to py and add `_()`

**Pull request merged**